### PR TITLE
MGMT-10919 - Use Pendo to track stats for vSphere

### PR DESCRIPTION
**Pull request by jgyselov**
https://issues.redhat.com/browse/MGMT-10919

We would like to start tracking vSphere-related controls in Assisted Installer, starting with the "Integrate with vSphere" switch and info-popover.

### CODE REVIEW

**Comment by jkilzi** [comment]
Needs https://github.com/openshift-assisted/assisted-ui-lib/pull/1505

**Comment by jkilzi** [comment]
LGTM

**Review by ryelo — CHANGES_REQUESTED** [review]
Trailing spaces are failing lint

File: data/openshift.yml [hunk 1]
```diff
@@ -415,3 +415,9 @@ openshift:
     "Subscriptions - Button: Take a tour":
       selectors:
         - '{}[data-ouia-app-id="subscriptions"] .pf-c-label__content:contains("Take a tour")'
+    "Assisted - Button: vSphere info":
+      selectors:
+        - '{}[data-ouia-app-id="assisted-installer"] [data-ouia-component-id="platform-integration-vSphere-popover"]'
+    "Assisted - Button: vSphere switch":
+      selectors:
```
Comment (jkilzi) [inline data/openshift.yml:422]:
```suggestion
      selectors:
```

Comment (jkilzi) [inline data/openshift.yml:422]:
```suggestion
      selectors:
```

**Review by jkilzi — COMMENTED** [review]
@jgyselov PTAL, there is a trailing white space

File: data/openshift.yml (hunk 1 above)
Comment (jkilzi) [reply data/openshift.yml:422]: @jgyselov this hasn't been solved yet the white space after the `:` is still there, line 422

**Review by jkilzi — CHANGES_REQUESTED** [review]

**Review by jkilzi — APPROVED** [review]

**Comment by jkilzi** [comment]
@ryelo FYI, trailing space removed

**Review by ryelo — APPROVED** [review]

**Pull request merged**